artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py

```diff
@@ -819,6 +819,8 @@
     "OpenNebula login panel was detected.": "wykryto panel logowania OpenNebula.",
     "Metabase login panel was detected.": "wykryto panel logowania Metabase.",
     "[no description] http/exposed-panels/fortinet/forticlientems-panel.yaml": "wykryto panel logowania FortiClient Endpoint Management Server.",
-    "Rancher Dashboard was detected.": "wykryto panel Rancher Dashboard.",
     "VMware Cloud Director Availability login panel was detected.": "wykryto panel logowania VMware Cloud Director Availability.",
+    "Rancher Dashboard was detected.": "wykryto panel systemu Rancher.",
+    "Mitric Checker login panel was detected.": "wykryto panel logowania Mitrix Checker.",
+    "Magento Connect Manager installer was detected. The software, available via /downloader/ location, requires Magento admin rights and uses the same authorization methods as for backend. If an attacker locates a matching pair of login/password, the installation will be compromised. An attacker can then discover backend URL for login (even if it is customized as described in Securing Magento  /admin/) and install a Filesystem extension to obtain full access to all files and finally the database.": "Wykryto panel Magento Connect Manager.",
```

Comment:
This is not quite `panel` as more `Wykryto ogólnodostępny instalator pod ścieżką /downloader/, poprzez który atakujący może odkryć działającą dane uwierzytelniające na konto administratora oraz dokonać zmian w aplikacji. Dzięki temu może on uzyskać dostęp od wszystkich plików oraz bazy danych aplikacji.`